packages/core/src/components/button/buttons.tsx

```diff
@@ -80,13 +88,46 @@ export class AnchorButton extends React.Component<React.HTMLProps<HTMLAnchorElem
             </a>
         );
     }
+
+    // Provide consistent keyboard interactions across both <Button /> and <AnchorButton /> (#430)
+    private onKeyDown = (e: React.KeyboardEvent<HTMLElement>) => {
+        const { href, onClick, target } = this.props;
+        if (e.which === Keys.SPACE) {
+            e.preventDefault();
+
+            if (href) {
+                if (target === undefined || target === "_self") {
+                    window.open(href);
+                } else if (target === "_blank") {
```

Comment:
but also there are other valid targets. did you try triggering real DOM `click()`? might be able to get all this behavior for free.

`ReactDOM.findDOMNode(this).click()`